.github/workflows/tests_and_coverage.yml

```diff
@@ -21,7 +25,7 @@ jobs:
 
     - name: CG Dependencies
       run: |
-        python -m pip install --upgrade pip==21.0.1
+        python -m pip install --upgrade pip
```

Comment:
will this adapt to the locked python version? Or perhaps pip version is never dependent on the python version?

Reply:
It never did hehe. And now its even breaking new coveralls if its too outdated
We locked it this way also because pip version wasn't allowed to be updated on clinical-db

Reply:
nice 👍